AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-search/vho-search.component.ts

```diff
@@ -47,14 +47,15 @@ export class VhoSearchComponent implements OnInit {
                         result = await this.service.getNonWorkAvailabilityForVho(this.username.value);
                         break;
                 }
-
-                if (result) {
-                    this.vhoSearchEmitter.emit(result);
+                var myItems = result.slice(0, 20);
```

Comment:
We'll need a unit test to check that it emits a maximum of 20 items.

Reply:
fixed the issues